rapid7_insight_agent/plugin.spec.yaml

```
@@ -0,0 +1,191 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: rapid7_insight_agent
+title: Rapid7 Insight Agent
+description: Take actions on an Insight Agent
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/rapid7_insight_agent
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- rapid7
+- agent
+- insight
+- edr
+- endpoint
+- detection
+- response
+- idr
+- ivm
+hub_tags:
+  use_cases: [threat_detection_and_response]
+  keywords: [rapid7, agent, endpoint, response, detection, insight, idr, ivm]
+  features: []
+types:
+  attributes:
```

```suggestion
  attribute:
```
Singular name since it refers to a single key:value pair

---

Can't change this, it's how it pops out in the JSON.